packages/react-native-web/src/vendor/react-native/VirtualizedList/index.js

```diff
@@ -1203,6 +1203,12 @@ class VirtualizedList extends React.PureComponent<Props, State> {
 
   _defaultRenderScrollComponent = props => {
     const onRefresh = props.onRefresh;
+    const inversionStyle = this.props.inverted
+    ? this.props.horizontal
+      ? styles.rowReverse
+      : styles.columnReverse
+    : null;
+
```

Comment:
Can we update these nested ternarys to be a little more readable with if/else?

Reply:
That's what RN Codebase has already been doing. Following their guidelines and approach here. 
https://github.com/Expensify/react-native-web/blob/947191f0fd5d43fcb5731e60fc3652e40fc7315d/packages/react-native-web/src/vendor/react-native/VirtualizedList/index.js#L880-L884

Reply:
Cool cool 👍  makes sense